bumble/core.py

```diff
@@ -209,13 +212,16 @@ def parse_uuid_2(cls, uuid_as_bytes, offset):
         return offset + 2, cls.from_bytes(uuid_as_bytes[offset : offset + 2])
 
     def to_bytes(self, force_128=False):
-        if len(self.uuid_bytes) == 16 or not force_128:
+        '''
+        Serialize UUID in little-endian byte-order
+        '''
+        if not force_128:
             return self.uuid_bytes
 
-        if len(self.uuid_bytes) == 4:
-            return self.uuid_bytes + UUID.BASE_UUID
-
-        return self.uuid_bytes + bytes([0, 0]) + UUID.BASE_UUID
+        match len(self.uuid_bytes):
```

Comment:
Unfortunately, `match` is a >= 3.10 feature, but we still need to support >= 3.8.